Automattic/jetpack
# Scheduled Updates: Read status from logs directly

Fixes Automattic/dotcom-forge#6551

## Proposed changes:

* This PR ensures we read the status from the logs directly instead of setting and reading from the `jetpack_scheduled_update_statuses` option.
*  When a user updates a schedule, we will inherit the logs as well since we consider it the same schedule.
* Remove all `set_scheduled_update_status` call. The actual function and status endpoint will be removed in the upcoming PR.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this PR by using Jetpack beta tester or follow the instructions below to `rsync` the changes to your testing WoA site.
* Navigate to `https://wordpress.com/plugins/scheduled-updates/<site-slug>` and make sure the status shows up correctly as before.
* Try to update your schedule and see if the logs get inherited.
* Try to remove and create a schedule and see if everything works as expected like before.
* Try to run CLI scheduled updates and see if the latest status gets reflected.
* Run the unit test and make sure nothing is broken.
